imagedephi/redact.py

```diff
@@ -67,8 +135,11 @@ def __init__(
                 # End of iteration, without "break"; no matching rule found anywhere
                 self.no_match_tags.append(tag)
 
+    def is_comprehensive(self) -> bool:
+        return len(self.no_match_tags) == 0
+
     def report_missing_rules(self) -> None:
-        if self.no_match_tags:
+        if not self.is_comprehensive():
             click.echo("The following tags could not be redacted given the current set of rules.")
             for tag in self.no_match_tags:
                 click.echo(f"{tag.value} - {tag.name}")
```

Comment:
`{tag.name}` is printing as a numerical value for me, so I see:
```
The following tags could not be redacted given the current set of rules.
305 - 305
```

Reply:
dfb3610 makes use of this parameter.